### PR TITLE
Limiter update

### PR DESCRIPTION
**Pull request by GolnesaK**

### CODE REVIEW

File: source/h_simulation_settings.rst [hunk 1]
```diff
@@ -119,30 +119,47 @@ Function where the ratio between water depth and water level gradient prescribes
...
 
-In this case the limiter is more or less effective depending of the local depth. In case the depth at the edge base on the down wind water level is larger than the definition that is given of a thin layer, the cross-sectional area is based on the high resolution grid. When this 'down wind' depth is smaller than the thin layer definition, then the limiter described for option 2 is determining the cross-sectional area. In the in between phase the two types of cross-sections are weighed to define a new value.
+The limiter *type 3* provides a smooth transition from the default water depth to the altered one. This transistion depends on the local depth and a user-defined "thin water layer" definition. In case the depth at the edge based on the downstream water level is larger than the thin layer definition, the cross-sectional area is based on the uniform water level assumption. In case the downstream water level is below the thin water layer definition, then limiter 2 determines the cross-sectional area. Finally, if the downstream water level is within the thin water layer depth, these two types of cross-sections are weighed to define the new value (i.e., limiter type 3).
```
Comment (nvolp) [inline source/h_simulation_settings.rst:136]: Can you elaborate a little bit more here. I think it needs a little bit of extra sentences to explain that the depth at the edge can be computed based on the up or downstream water level.

File: source/h_simulation_settings.rst [hunk 2]
```diff
@@ -119,30 +119,47 @@ Function where the ratio between water depth and water level gradient prescribes
...
+   Water distribution based on uniform water level assumption showing x-z and x-y profiles
+
+.. figure:: image/limiter1.png
+   :figwidth: 1000 px
+   :alt: limiter_1
+
+   Water distribution based on limiter 1 showing x-z and x-y profiles
+
+.. figure:: image/limiter2.png
+   :figwidth: 1000 px
```
Comment (nvolp) [inline source/h_simulation_settings.rst:153]: Beautifull figures !!!
Can we distribute them in the text as close as possible to the explainations?

**Review by nvolp — CHANGES_REQUESTED** [review]
Great! It is so much better than it was! It made some small changes. And two minor requests, which are harder to do without checking out the full code.

File: source/h_simulation_settings.rst (hunk 1 above)
Comment (GolnesaK) [reply source/h_simulation_settings.rst:136]: I don't understand...There is already a sentence indicating that the depth at edge based on downstream is compared to the thin-layer.

**Review by nvolp — APPROVED** [review]

**Pull request merged**